<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
    font-family: &quot;Segoe UI&quot;;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="ShowDataTable">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>864</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Load"/>
    <addaction name="action_Save"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuTime">
    <property name="title">
     <string>&amp;Time</string>
    </property>
    <addaction name="action_Start"/>
    <addaction name="action_Stop"/>
   </widget>
   <widget class="QMenu" name="menuHel">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_How_to_use"/>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionAdd_Data_Pane"/>
    <addaction name="action_Table"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuTime"/>
   <addaction name="menu_View"/>
   <addaction name="menuHel"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="AddDataDockWindow">
   <property name="font">
    <font>
     <family>Segoe UI</family>
    </font>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Work Time Pane</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="AddDataDockContents">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QScrollArea" name="AddDataScrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="AddDataScrollContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>204</width>
          <height>383</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="NumberLabel">
             <property name="text">
              <string>Number</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="CategoryLabel">
             <property name="text">
              <string>Category</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="NumberLineEdit">
             <property name="inputMask">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="DescriptionLineEdit">
             <property name="placeholderText">
              <string>Detail</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="DescriptionLabel">
             <property name="text">
              <string>Description</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="CategoryLineEdit">
             <property name="placeholderText">
              <string>Project</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QRadioButton" name="CurrentRadioButton">
             <property name="text">
              <string>Current Time</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLineEdit" name="TimeLineEdit">
             <property name="placeholderText">
              <string>HH:MM</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0" colspan="2">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="5" column="0" colspan="2">
            <widget class="QRadioButton" name="CustomRadioButton">
             <property name="text">
              <string>Custom Time</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="TimeLabel">
             <property name="text">
              <string>Time</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="DateLabel">
             <property name="text">
              <string>Date</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="StartButton">
               <property name="text">
                <string>Start</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="StopButton">
               <property name="text">
                <string>Stop</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="7" column="1">
            <widget class="QLineEdit" name="DateLineEdit">
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>yyyy-mm-dd</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Load">
   <property name="text">
    <string>&amp;Load</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Start">
   <property name="text">
    <string>&amp;Start</string>
   </property>
  </action>
  <action name="action_Stop">
   <property name="text">
    <string>&amp;Stop</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_How_to_use">
   <property name="text">
    <string>&amp;How to use</string>
   </property>
  </action>
  <action name="actionAdd_Data_Pane">
   <property name="text">
    <string>&amp;Add Data Pane</string>
   </property>
  </action>
  <action name="action_Table">
   <property name="text">
    <string>&amp;Table</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>NumberLineEdit</tabstop>
  <tabstop>CategoryLineEdit</tabstop>
  <tabstop>DescriptionLineEdit</tabstop>
  <tabstop>CurrentRadioButton</tabstop>
  <tabstop>CustomRadioButton</tabstop>
  <tabstop>DateLineEdit</tabstop>
  <tabstop>TimeLineEdit</tabstop>
  <tabstop>StartButton</tabstop>
  <tabstop>StopButton</tabstop>
  <tabstop>ShowDataTable</tabstop>
  <tabstop>AddDataScrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
